src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/EqualConverter.shared.cs

```diff
@@ -18,8 +18,10 @@ public class EqualConverter : ValueConverterExtension, IValueConverter
 		/// <param name="parameter">The second object to compare.</param>
 		/// <param name="culture">The culture to use in the converter. This is not implemented.</param>
 		/// <returns>True if <paramref name="value"/> and <paramref name="parameter"/> are equal, False if they are not equal.</returns>
-		public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
-			=> (value != null && value.Equals(parameter)) || (value == null && parameter == null);
+		public object Convert(object value, Type targetType, object parameter, CultureInfo culture) => ConvertInternal(value, parameter);
+
+		internal static bool ConvertInternal(object value, object parameter) =>
```

Comment:
Why is it static?

It can be protected and instace

Reply:
Initial solution was [exactly like you describe](https://github.com/xamarin/XamarinCommunityToolkit/pull/935/commits/62c43e1a34699381486ea4e9ea477b337b0dd019), but @pictos wants convertors to be sealed to  avoid issue with inheritance. I think that discussion got deleted, I cannot find it. So we changed fix to use static methods.

Reply:
@maxkoshevoi also with inheritance didn't work, right? Or am I mix with another issue?

Reply:
@pictos It did. 
I just tried to do it without `virtual` using `new` instead of `override` after your request to not use `virtual` (just wanted to see if it would work), and that didn't work. But `virtual` - `override` worked as expected.

Reply:
Got ya. And yeah, I want to avoid inheritance, since isn't a good practice in places that we don't expect it.